packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/CommunityStakeStep/EnableStake/EnableStake.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import { z } from 'zod';
+
+import { CWDivider } from 'views/components/component_kit/cw_divider';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWForm } from 'views/components/component_kit/new_designs/CWForm';
+import { CWTextInput } from 'views/components/component_kit/new_designs/CWTextInput';
+import { CWButton } from 'views/components/component_kit/new_designs/cw_button';
+import CreateCommunityHint from '../../../components/CreateCommunityHint';
+
+import './EnableStake.scss';
+
+const validationSchema = z.object({
+  namespace: z.string().min(1, 'Namespace is required'),
+  symbol: z
+    .string()
+    .min(1, 'Symbol is required')
+    .max(9, 'Symbol should be maximum 9 characters'),
+});
```

Comment:
suggestion: we move this to a `validations.ts` file in same folder

Reply:
moved